benches/fragmentation/synthetic/create_fragmentation_in_pools/create_fragmentation_in_pools.jl

```diff
@@ -0,0 +1,111 @@
+@static if VERSION < v"1.12.0-DEV.0"
+    error("This script requires Julia 1.12 or later")
+end
+@static if Sys.WORD_SIZE != 64
+    error("This script requires a 64-bit version of Julia")
+end
+@static if !Sys.isunix()
+    error("This script requires a Unix-like operating system")
+end
+const SIZE_CLASSES_FROM_JL_STOCK_GC = vcat(
```

Comment:
is there a way we can look this up rather than being dependent on implimentation details?